textdistance/libraries.json

```diff
@@ -19,39 +15,39 @@
       "hamming"
     ],
     [
-      "jellyfish",
-      "hamming_distance"
-    ],
-    [
-      "distance",
-      "hamming"
+      "rapidfuzz.distance.hamming",
+      "distance"
     ],
     [
-      "abydos.distance",
```

Comment:
Did you remove abydos from the list? Why? Is this file autogenerated? I don't remember 🤔  Why is it not in the list anymore, do you have an idea?

Reply:
I think this was generated and I did not have abydos on my machine at the time (abydos still fails on python3.10 for me). Manually added them back